docs/current_docs/integrations/snippets/depot-test-build.yml

```diff
@@ -0,0 +1,43 @@
+name: dagger
+on:
+  push:
+    branches: [main]
+
+jobs:
+  test:
+    name: test
+    runs-on: depot-ubuntu-latest-*,dagger=0.13.5
```

Comment:
`-*` is meant to signal your can choose your number of CPUs? I didn't see this called out anywhere in the docs.

Reply:
I wasn't sure what the `*` referred to, I used the syntax shown on https://depot.dev/docs/github-actions/runner-types. I've changed it to be more specific now.